Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/setup.py

```diff
@@ -481,6 +481,8 @@ def parser(cls, parser, loggers=None):
 
     @classmethod
     def main(cls, args, repository, **kwargs):
+        import jinja2
```

Comment:
How does importing more things fix the mentioned issue?

Reply:
We're already import `jinja2` The difference is, by importing it here, we do so before we prompt the user for anything.

Reply:
ok. I was hoping the solution would be result in no modules being installed during git webkit setup, and probably be installed earlier in some manner. But I guess there isn't a straight-forward way to do that.
 
Also, what about MarkupSafe (mentioned in https://bugs.webkit.org/show_bug.cgi?id=240487#c0)?

Also, fixing https://bugs.webkit.org/show_bug.cgi?id=229865 would also make it less annoying to users.

Reply:
`MarkupSafe` is a dependency of `jinja2`.

And if contributors have run Python commands in a checkout before, then `git-webkit setup` _won't_ install any modules. @aproskuryakov had, if I'm not mistaken, had a totally pristine checkout.